testing-common/src/main/java/io/opentelemetry/instrumentation/testing/LibraryTestRunner.java

```diff
@@ -44,7 +46,7 @@ public final class LibraryTestRunner extends InstrumentationTestRunner {
     GlobalOpenTelemetry.resetForTest();
 
     testSpanExporter = InMemorySpanExporter.create();
-    testMetricExporter = InMemoryMetricExporter.create();
+    testMetricExporter = InMemoryMetricExporter.create(AggregationTemporality.DELTA);
```

Comment:
Cumulative metrics are a bad fit for tests. We have a shard instance of the SDK for all tests. If temporality is cumulative, then metrics that initially received measurements for one test will continue to report for the remaining tests. The result is very cumbersome to assert against and debug.